keyboards/mmkeyboard/class60/soldered/soldered.c

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "quantum.h"
+
+#ifdef AUDIO_ENABLE
+float caps_on[][2] = SONG(CAPS_LOCK_ON_SOUND);
+float caps_off[][2] = SONG(CAPS_LOCK_OFF_SOUND);
+
+float num_on[][2] = SONG(NUM_LOCK_ON_SOUND);
+float num_off[][2] = SONG(NUM_LOCK_OFF_SOUND);
+
+float scroll_on[][2] = SONG(SCROLL_LOCK_ON_SOUND);
+float scroll_off[][2] = SONG(SCROLL_LOCK_OFF_SOUND);
+
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if (res) {
```

Comment:
```suggestion
    if (res) {
        led_update_ports(led_state);
```

Reply:
Thanks, that's a nice and simple solution.

Reply:
Actually, looking at the current uses of that function: https://github.com/search?q=repo%3Aqmk%2Fqmk_firmware+led_update_ports+path%3A%2F%5Ekeyboards%5C%2F%2F&type=code it looks like this is just a callback that can be implemented in a keyboard implementation. Not sure if calling this here will help.

Reply:
I guess this should be fine looking at: https://github.com/qmk/qmk_firmware/blob/master/quantum/led.c#L84-L90. Guess there was no similar use-case before.